### PR TITLE
[AIRFLOW-2853] Add official committers to README

### PR DESCRIPTION
**Pull request by andscoop**
Make sure you have checked _all_ steps below.

### Jira

- [x] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
  - https://issues.apache.org/jira/browse/AIRFLOW-2853
  - In case you are fixing a typo in the documentation you can prepend your commit with \[AIRFLOW-XXX\], code changes always need a Jira issue.

### Description

- [x] Here are some details about my PR, including screenshots of any UI changes:

This PR brings a list of maintainers into the code repo so that they can easily be found and live with the repo itself.

### Tests

- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

Docs

### Commits

- [x] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [x] In case of new functionality, my PR adds documentation that describes how to use it.
  - When adding new operators/hooks/sensors, the autoclass documentation generation needs to be added.

### Code Quality

- [x] Passes `git diff upstream/master -u -- "*.py" | flake8 --diff`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -274,6 +270,35 @@ Currently **officially** using Airflow:
...
+1. Dan Davydov
+1. [Fokko Driesprong](https://github.com/fokko)
+1. [Jeremiah Lowin](https://github.com/jlowin)
+1. [Joy Gao](https://github.com/joygao)
+1. [Kaxil Naik](https://github.com/kaxil)
+1. [Maxime Beauchemin](https://github.com/mistercrunch)
+1. [Patrick Leo Tardif](https://github.com/patrickleotardif)
+1. [Siddharth Anand](https://github.com/r39132)
+1. Steven Yvinec-Kruyk
+1. [Sumit Maheshwari](https://github.com/msumit)
```
Comment (ashb) [inline README.md]: This list is slightly out of date - you are missing some people, and Alex Guziel isn't a commiter? - the authoritative list is http://people.apache.org/committers-by-project.html#airflow

**Review by ashb — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (andscoop) [reply README.md]: @ashb Perfect, I was looking for this and was unable to find it. I was going off of https://incubator.apache.org/projects/airflow.html . Will update

Comment (andscoop) [reply README.md]: @ashb I have updated according to that list, just a note, I left Alex on as he is on the list you linked to.

Comment (ashb) [reply README.md]: Oh so he is - it's just sorted by user id :)

**Review by ashb — APPROVED** [review]

**Comment by r39132** [comment]
@andscoop In your Jira, you referenced creating a CODEOWNERS files. Why is that not the way to proceed?

**Comment by andscoop** [comment]
@r39132 I did not originally research thoroughly. 

It would require us to set owners to various sections of the codebase so that those owners are prompted for automatic review when a PR for that section is submitted. CODEOWNERS may be the direction the project wants to go eventually - but I don't believe it (the project) currently goes into the level of detail of accepting committers for specific sections of the codebase. While some committers may be more inclined to review PRs from a specific section, from the outside looking in, it appears that most committers/maintainers float across various sections of the codebase.

**Comment by r39132** [comment]
Thx @andscoop. I agree we are not there yet.

Ok, so I sent a note to the dev list that we have 3 places currently where the committer list is maintained : https://lists.apache.org/thread.html/15cf9c3f0d45a08f988fe9f3ff0fa6092ef8a8655e6087177fbc4a51@%3Cdev.airflow.apache.org%3E

Merging this PR would require yet another list to be updated. This is why I originally linked via the cwiki. If you look at the cwiki right now, I did add all 3 places where committer lists are maintained. Does this satisfy your need?

**Comment by r39132** [comment]
I do like what you have written though and would love you to alter the cwiki page. I was trying to add you an admin to the cwiki, but could not find you for some reason. I tried to add andscoop and Andy Cooper. if you go by some other name, do let me know.

**Comment by andscoop** [comment]
@r39132 please try again - just officially signed up for the cwiki which is apparently different than the jira signup.

I understand the concern with maintaining this is 3 different places. Main reason for the PR is IMO that docs which don't live with the code get left behind. I will gladly update the wiki if we choose not to move forward with the PR.

**Comment by r39132** [comment]
@andscoop I granted you access... though I see in your PR, that you do still link to the cwiki. You may not have much to update in the wiki itself.

I do like the "Who maintains Airflow" section that you added. I would be open to merging this PR if you didn't have the explicit list of committers listed .. we have enough PRs to merge as it is and there a quite a few steps involved in promoting a contributor to committer status -- I'd be happy not to add yet another step to it. 

That said, you could add a link to https://github.com/orgs/apache/teams/airflow-committers/members on the read me so people could see the list of committers in a single hop. 

Would this work for you?

**Comment by andscoop** [comment]
@r39132 Looks like a couple of these links need [team access to github](https://github.com/orgs/apache/teams/airflow-committers/members) and access to [whimsy server](https://whimsy.apache.org/roster/ppmc/airflow). So we can't put those in the file. The best definitive resource I have seen so far is the one @ashb [linked to in review](https://people.apache.org/committers-by-project.html#airflow). Is this the link we should move forward with?


edit - removed list of contributed and updated with list of known working links

**Comment by r39132** [comment]
LGTM!

**Pull request closed without merging**